src/webgl/p5.RendererGL.js

```diff
@@ -1874,6 +1905,85 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     this.textures.set(src, tex);
     return tex;
   }
+  /*
+    *  used in imageLight,
+    *  To create a blurry image from the input non blurry img,
+    *  Add it to the blurryTexture map,
```

Comment:
I think we renamed `blurryTexture` to `diffusedTextures`. Also we can mention that we only create it if it does not already exist.